.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Pull request
+on:
+  release:
```

Comment:
I don't think this will work—releases are created by the `Run semantic-release` step, so there's a circular dependency here. This workflow should run on any push to `master`.

Reply:
changed event to be on push